addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -524,57 +494,51 @@ def _import_fattura_pa(self, tree, invoice):
+                            discount_elements = element.xpath('.//ScontoMaggiorazione')
+                            if discount_elements:
+                                discount_element = discount_elements[0]
+                                discount_percentage = discount_element.xpath('.//Percentuale')
+                                if discount_percentage and len(discount_elements) == 1:
+                                    discount_type = discount_element.xpath('.//Tipo')
+                                    discount_sign = 1 
+                                    if discount_type and discount_type[0].text == 'MG':
+                                        discount_sign = -1
+                                    invoice_line_form.discount = discount_sign * float(discount_percentage[0].text)
+                                else:
+                                    total = float(element.xpath('.//PrezzoTotale')[0].text)
+                                    discount = 100 - (100 * total) / (invoice_line_form.quantity * invoice_line_form.price_unit)
+                                    invoice_line_form.discount = discount
+
+
+                # Global discount summarized in 1 amount
+                discount_elements = body_tree.xpath('.//DatiGeneraliDocumento/ScontoMaggiorazione')
+                if discount_elements:
+                    taxable_amount = sum([float(amount.text) for amount in body_tree.xpath('.//ImponibileImporto')])
+                    tax_amount = sum([float(amount.text) for amount in body_tree.xpath('.//Imposta')])
+                    total_amount = taxable_amount + tax_amount
+                    invoice_total_amount = total_amount
+                    for discount_element in discount_elements:
+                        discount_type = discount_element.xpath('.//Tipo')
+                        discount_sign = 1 
+                        if discount_type and discount_type[0].text == 'MG':
+                            discount_sign = -1
+                        discount_amount = discount_element.xpath('.//Importo')
+                        if discount_amount:
+                            invoice_total_amount -= float(discount_amount[0].text)
+                            continue
+                        discount_percentage = discount_element.xpath('.//Percentuale')
+                        if discount_percentage :
+                            invoice_total_amount *= 1 - float(discount_percentage[0].text) / 100
+
+                    general_discount = invoice_total_amount - taxable_amount - tax_amount
+                    sequence = len(elements) + 1
+
+                    with invoice_form.invoice_line_ids.new() as invoice_line_global_discount:
+                        invoice_line_global_discount.tax_ids.clear()
+                        invoice_line_global_discount.sequence = sequence
+                        invoice_line_global_discount.name = 'GLOBAL DISCOUNT' if general_discount < 0 else 'GLOBAL EXTRA CHARGE'
```

Comment:
I would say: use _( to translate, but towards a non-translated field...  Or we just add it in Italian: Sconto:  Maggiorazione: (if I am not mistaken)  Or we use the _( as we did before.